docs/distributed/quick-start-guide.md

```diff
@@ -0,0 +1,78 @@
+# Distributed Minio Quickstart Guide [![Gitter](https://badges.gitter.im/Join%20Chat.svg)](https://gitter.im/minio/minio?utm_source=badge&utm_medium=badge&utm_campaign=pr-badge&utm_content=badge) [![Go Report Card](https://goreportcard.com/badge/minio/minio)](https://goreportcard.com/report/minio/minio) [![Docker Pulls](https://img.shields.io/docker/pulls/minio/minio.svg?maxAge=604800)](https://hub.docker.com/r/minio/minio/) 
+
+Minio in distributed mode lets you pool multiple drives (even on different machines) into a single object storage server. As drives are distributed across several nodes, distributed Minio can withstand multiple node failures and yet ensure full data protection.
+
+## Why distributed Minio?
+
+Minio in distributed mode can help you setup a highly-available storage system with a single object storage deployment. With distributed Minio, you can optimally use storage devices, irrespective of their location in a network.
+
+### Data protection
+
+Distributed Minio provides protection against multiple node/drive failures and [bit rot](https://github.com/minio/minio/blob/master/docs/erasure/README.md#what-is-bit-rot-protection) using [erasure code](https://docs.minio.io/docs/minio-erasure-code-quickstart-guide). As the minimum disks required for distributed Minio is 4 (same as minimum disks required for erasure coding), erasure code automatically kicks in as you launch distributed Minio.
+
+### High availability
+
+A stand-alone Minio server would go down if the server hosting the disks goes offline. In contrast, a distributed Minio setup with _n_ disks, would be online as long as [Quorum](https://github.com/minio/dsync#lock-process) _(n/2 + 1)_ or more disks are online. 
+
+For example, a 8 nodes 8 disks (one disk per node) distributed Minio setup would stay put, even if upto 3 nodes are offline.
+
```

Comment:
We should clarify that data is safe even when four nodes are down (n/2) but for new object creation we will need min of 5 nodes online (n/2 +1)...